src/content/drafts/introducing-bulk-suppressions.md

```diff
@@ -31,10 +31,15 @@ This approach prioritizes preventing new violations while providing a clear path
 
 Once you've configured a new rule as `"error"` in your ESLint configuration, you can bulk suppress existing violations using the command line. You can choose to suppress all violations or specific rules.
 
+Please note that it's recommended to run the suppression command with the `--fix` flag to automatically resolve any violations that can be fixed.
+
 ```
 # Suppress all violations
```

Comment:
Should we remove this line?

Reply:
I think not because it provides a short description for each command. Keeping it consistent.

Reply:
My concern is that immediately before it we're telling people to use `--fix` and then showing an example where it's not using `--fix`. It seems like one of these two needs to change to be consistent.

Reply:
Clear. I have removed the examples without --fix